app/models/move.rb

```diff
@@ -126,17 +126,22 @@ def rebooked
   end
 
   def rebook
-    rebooked || self.class.create(
+    return rebooked if rebooked.present?
+
+    new_booking = self.class.new(
       original_move_id: id,
       from_location_id: from_location_id,
       to_location_id: to_location_id,
       allocation_id: allocation_id,
       profile_id: profile_id,
       status: MOVE_STATUS_PROPOSED,
       date: date && date + 7.days,
-      date_from: date_from && date_from + 7.days,
+      date_from: (date_from || date) + 7.days,
```

Comment:
```suggestion
      date_from: (date || date_from) + 7.days,
```

We should use the move date first because that's the confirmed date

Reply:
are you certain that changing the behaviour of rebooking a move to be based on `date` rather than `date_from` is sensible? I think that might be wiser to tackle that in its own issue rather than as part of this.

Reply:
I think we should revert to the original, yeah.